SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security Policy
+
+## Supported Versions
+
+scikit-image is generally backwards compatible with few exceptions, so we
+recommend users to always use the latest version.
+
+We usually don't backport patches or fixes to previous versions. But we may
+release minor versions with patches. In case of particular critical issues
+exceptions may be made.
+
+## Reporting a Vulnerability
+
+If you believe you've discovered a serious vulnerability, please contact the
+scikit-image core team at: <br>
+skimage-core@discuss.scientific-python.org
+
+We will evaluate your report and if necessary issue a fix and an advisory. If
+the issue was previously undisclosed, we'll also mention your name in the
+release notes.
+
+## Responsible Disclosure
+
+We may apply a responsible disclosure process to reported or otherwise
```

Comment:
What does this mean? Looking on the internet, it seems like that means we would provide companies using our software early warnings before we announce the issue publicly.